docs/source/blackbox/score_decrease.rst

```diff
@@ -0,0 +1,91 @@
+.. _eli5-decrease-in-score:
+
+Decrease in Score
+=================
+
+eli5 provides a way to compute feature importances for any black-box
+estimator by measuring how score decreases when a feature is not available.
+
+A similar method is described in Breiman, "Random Forests", Machine Learning,
+45(1), 5-32, 2001 (available online at
+https://www.stat.berkeley.edu/%7Ebreiman/randomforest2001.pdf) as
```

Comment:
Yeah, I should be more careful with literature. Method is described on pages 18-19, but it is not called "mean decrease in accuracy" there. It is called "mean decrease in accuracy" here: https://dinsdalelab.sdsu.edu/metag.stats/code/randomforest.html and here: http://alexperrier.github.io/jekyll/update/2015/08/27/feature-importance-random-forests-gini-accuracy.html, and in several papers like this http://ieeexplore.ieee.org/document/7883053/?reload=true. It is also called "Mean Decrease Accuracy" here: http://www.montefiore.ulg.ac.be/%7Eglouppe/pdf/phd-thesis.pdf (see page 125), saying it is proposed in Brieman's papers:

> In addition to MDI, Breiman [2001,2002] also proposed to evaluate the importance of a variable Xj by measuring the Mean Decrease Accuracy (MDA) or, equivalently, by measuring the Mean Increase Error, of the forest when the values of Xj are randomly permuted in the out-of-bag samples. For that reason, this latter measure is also known as the permutation  importance. 

So yeah, I should find a better reference, and at least mention the alternative name (permutation importance) which is also common.